installer/src/main/java/com/amazon/aws/partners/saasfactory/saasboost/SaaSBoostInstall.java

```diff
@@ -297,16 +309,245 @@ protected void installSaaSBoost(String existingBucket) {
+
+            List<CertificateSummary> certificates = existingCertificates(acm, identityProviderCustomDomain);
+            if (certificates == null || certificates.isEmpty()) {
+                System.out.println("Error. Unable to find an ACM certificate for the domain "
+                        + identityProviderCustomDomain + ". Create or import a public certificate in this Region.");
+                cancel();
+            } else {
+                while (true) {
+                    System.out.println("Select the ACM Certificate for the domain "
+                            + identityProviderCustomDomain + ":");
+                    for (ListIterator<CertificateSummary> iter = certificates.listIterator(); iter.hasNext(); ) {
+                        int index = iter.nextIndex();
+                        CertificateSummary certificate = iter.next();
+                        System.out.printf("%d. %s%n",
+                                (index + 1),
+                                certificate.domainName()
+                        );
+                    }
+                    System.out.print("Type the number of the certificate to use and press enter: ");
+                    Integer choice = Keyboard.readInt();
+                    try {
+                        identityProviderCertificate = certificates.get((choice - 1)).certificateArn();
+                        LOGGER.info("Setting identity provider certificate = [{}]", identityProviderCertificate);
+                        break;
+                    } catch (NullPointerException | IndexOutOfBoundsException e) {
+                        outputMessage("Invalid choice, please try again. Enter the number of the certificate to use.");
+                    }
+                }
+            }
+        }
+
+        boolean useCustomDomainForAdminWebApp = Utils.isChinaRegion(AWS_REGION);
+        if (!useCustomDomainForAdminWebApp) {
+            System.out.print("Would you like to use a custom domain name for the SaaS Boost admin web console (y or n)? ");
+            useCustomDomainForAdminWebApp = Keyboard.readBoolean();
+        }
+        String adminWebAppCustomDomain = null;
+        String adminWebAppHostedZone = null;
+        String adminWebAppCertificate = null;
+        if (useCustomDomainForAdminWebApp) {
+            if (!Utils.isChinaRegion(AWS_REGION)) {
+                System.out.println("You must provide a verified TLS (SSL) certificate from ACM in the us-east-1 "
+                        + " region to use a custom domain name for the SaaS Boost admin web console.");
+            } else {
+                System.out.println("You must provide a verified TLS (SSL) certificate from the IAM certificate store "
+                        + " to use a custom domain name for the SaaS Boost admin web console.");
+                System.out.println("You must also have an ICP registration for the domain. See https://www.amazonaws.cn/en/about-aws/china/#ICP_ for more information.");
```

Comment:
Looks like this link should be `https://www.amazonaws.cn/en/about-aws/china/#ICP_in_China`